editor/editor_node.cpp

```diff
@@ -1393,7 +1394,8 @@ void EditorNode::_set_scene_metadata(const String &p_file, int p_idx) {
 		return;
 	}
 
-	scene->set_meta("__editor_run_settings__", Variant()); // clear it (no point in keeping it)
+	// Clear meta value (no point in keeping it).
+	scene->set_meta("__editor_run_settings__", Variant());
 	scene->set_meta("__editor_plugin_states__", Variant());
```

Comment:
This could be changed to `remove_meta()`

Reply:
Just did a quick search and it looks like this is actually the only place where these meta values are used, so I guess we could remove them completely.

Reply:
Indeed:
```
$ rg __editor_run_settings__
editor/editor_node.cpp
1396:   scene->set_meta("__editor_run_settings__", Variant()); // clear it (no point in keeping it)
$ rg __editor_plugin_states__
editor/editor_node.cpp
1397:   scene->set_meta("__editor_plugin_states__", Variant());
```

Reply:
Removed it.